static/usage/v7/fab/safe-area/angular/example_component_html.md

```diff
@@ -0,0 +1,7 @@
+```html
+<ion-fab slot="fixed" vertical="top" horizontal="center">
+  <ion-fab-button>
+    <ion-icon name="add"></ion-icon>
```

Comment:
The icons in the angular and javascript examples don't load in stackblitz because of #2892